heroku/templates/src/main/java/package/config/_HerokuDatabaseConfiguration.java

```diff
@@ -32,32 +32,22 @@ public void setEnvironment(Environment environment) {
     public DataSource dataSource(<% if (hibernateCache == 'hazelcast') { %>CacheManager cacheManager<% } %>) {
         log.debug("Configuring Heroku Datasource");
 
-        String herokuUrl = propertyResolver.getProperty("heroku-url");
+        String herokuUrl = System.getenv("JDBC_DATABASE_URL");
```

Comment:
I'm not sure about the "System.getenv" -> we could do the same with the Spring propertyResolver, normally, and this is how we do it everywhere else